cloud-tool/variables.tf

```diff
@@ -0,0 +1,125 @@
+######################################################################
+# Copyright (c) 2021 Claudio André <claudioandre.br at gmail.com>
+#
+# This program comes with ABSOLUTELY NO WARRANTY; express or implied.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, as expressed in version 2, seen at
+# http://www.gnu.org/licenses/gpl-2.0.html
+######################################################################
+
+# TODO ###############################################################
+# Change the cidr_blocks below to update using your IP address.
+variable ingress_data {
+  description = "The secutiry groups inbound rules."
+  type = map(object({description = string, cidr_blocks = list(string)}))
+  default = {
+    22 = { description = "Inbound SSH rule", cidr_blocks = [ "45.170.58.160/32" ] }
```

Comment:
Hard-coded IP address here. (Also, typo in "secutiry" in at least two places.)

Reply:
I can switch to "YOUR_IP_HERE/32". I can't think of another good numerical value.